src/material/core/tokens/m2/mat/_select.scss

```diff
@@ -17,6 +17,7 @@ $prefix: (mat, select);
   $on-surface: if($is-dark, #fff, #000);
 
   @return (
+    list-item-selected-container-color: rgba($on-surface, if($is-dark, 0.08, 0.04)),
```

Comment:
In M3 the select arrow isn't aligned on the lower densities, but in M2 it is. Maybe we're missing a token somewhere? 
<img width="778" alt="image" src="https://github.com/angular/components/assets/4450522/83f149f9-f304-4174-9217-35bf3ab77546">

Reply:
Yeah, density for the select seems to be pretty broken atm. I was planning to address this in a follow-up PR

Reply:
That's fine. Resolving.